storage/s3_sdk/list_gcs_buckets.go

```diff
@@ -0,0 +1,56 @@
+package main
```

Comment:
Samples should be standalone snippets, not runnable binaries, unless they're a quickstart (part of the new rubric guidance).